presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -1235,6 +1246,12 @@ private static boolean containsAllPrivilege(Set<PrivilegeGrantInfo> requestedPri
     private RetryDriver retry()
     {
         return RetryDriver.retry()
+                .exponentialBackoff(
+                        this.minBackoffDelay,
```

Comment:
Please remove `this` from these field references as they aren't needed. We only use them when required, or sometimes for all assignments in the constructor (to be consistent with the assignments where it is required).